wordpress-mobile/WordPress-iOS
# Reader post comments: set custom button border color

Ref: #17511 

This adds functionality to `BorderedButtonTableViewCell` to specify a button border color. That is used for the `View All Comments` button to set the border grey.

To test:
- Enable the `postDetailsComments` feature.
- Go to Reader > post > details > Comments.
- Verify the border color is grey and the label color is still black.

https://user-images.githubusercontent.com/1816888/144127725-301fb7a0-7ed3-4389-a1a4-314cf3a27e70.mp4

https://user-images.githubusercontent.com/1816888/144127715-294fee86-17a8-4bee-9bfc-446924722113.mp4

## Regression Notes
1. Potential unintended areas of impact
N/A. WIP.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. WIP.

3. What automated tests I added (or what prevented me from doing so)
N/A. WIP.

PR submission checklist:
